tests/internals.rs

```diff
@@ -534,15 +576,11 @@ fn test_create_account_tx_test_state() {
     let class_hash = state.get_class_hash_at(&TEST_CONTRACT_ADDRESS).unwrap();
     assert_eq!(class_hash, felt_to_hash(&TEST_CLASS_HASH));
 
-    let contract_class: ContractClass = state
+    let _contract_class: ContractClass = state
         .get_contract_class(&felt_to_hash(&TEST_ERC20_CONTRACT_CLASS_HASH))
         .unwrap()
         .try_into()
         .unwrap();
-    assert_eq!(
```

Comment:
This is due to the `Program` comparison bug, right? I think we should comment it like the rest.

Reply:
Bono just asked me to remove it hahaha

Reply:
Ohh in that case we need to add a comment detailing why it is commented and what we need to do to uncomment it